Resources/config/services.xml

```diff
@@ -92,6 +92,9 @@
             <argument type="service" id="Symfony\Component\Security\Core\Security" />
             <tag name="kernel.event_listener" event="trikoder.oauth2.authorization_request_resolve" method="onAuthorizationRequest" priority="1024" />
         </service>
+        <service id="trikoder.oauth2.event_listener.authorization.convert_to_response" class="Trikoder\Bundle\OAuth2Bundle\EventListener\ConvertExceptionToResponseListener">
+            <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" priority="10" />
```

Comment:
@Nyholm Please make the priority configurable via the bundle extension. We've had already similar cases in our (legacy) apps where we had to override listener definitions from some bundles in our apps just to change the priority so I think it would be good to be flexible here.